curator-recipes/src/main/java/org/apache/curator/framework/recipes/nodes/GroupMember.java

```diff
@@ -75,6 +119,7 @@ public void start()
         try
         {
             cache.start();
+            cache.getListenable().addListener(childrenCacheListener);
```

Comment:
Why not add a proxy listener getListenable()? I need to think about this. Does this single listener correctly proxy all behaviors from getListenable()?